packages/rancher-logging/overlay/templates/outputs/elasticsearch/flow.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: logging.banzaicloud.io/v1beta1
```

Comment:
do the `if .Values.elasticsearch.enabled` need to be here since they were removed from the other template?

Reply:
Yes logging can be created regardless of of we have an output or not, the flow should only be created if an output exists for it. in this case this flow belongs to es and should only be created for es.

Reply:
Oh nvm, it already is